integrate/quad/hermite.go

```diff
@@ -273,7 +273,7 @@ func (h Hermite) hermiteInitialGuess(i, n int) float64 {
 	}
 
 	// Use Gatteschi guesses in the second half where x is nearer to sqrt(n+0.5)
-	i = i + 1 - m
+	i = m - i - 1
```

Comment:
Can you make this `i = m - (i + 1)` for clarity?

Reply:
Yes, of course